docker/model-server/release.sh

```diff
@@ -17,6 +17,10 @@ docker build --pull \
     -t bentoml/model-server:"$BENTOML_VERSION" \
     .
 docker push bentoml/model-server:"$BENTOML_VERSION"
+echo "Tag and push 'latest'.."
+docker tag bentoml/model-server:"$BENTOML_VERSION" bentoml/model-server:latest
+docker push bentoml/model-server:latest
```

Comment:
Thoughts on tagging with latest in the build step? https://github.com/moby/moby/pull/15780

Reply:
That way push step can just be `docker push bentoml/model-server` and it will figure out which tags need to be pushed on its own

Reply:
Great idea!

Do you know how does `docker push bentoml/model-server` figure out which tags needs to be pushed? I may have other tags that are used only for local development and want to avoid pushing those to docker hub.

Reply:
Ok turns out `docker push` without a specified tag will push all local dev images too so that may not be ideal. I guess we can just leave it at `docker push bentoml/model-server:latest && docker push bentoml/model-server:"$BENTOML_VERSION"`
![image](https://user-images.githubusercontent.com/23178940/91668509-1c71da80-eac2-11ea-9479-ffb153612435.png)

Reply:
That makes sense, thanks for confirming!